### PR TITLE
Add genome sequencing to NewCaseForm

### PR DESCRIPTION
**Pull request by allysonjp715**
Screen Shot 2020-06-26 at 10 11 41 AM

### CODE REVIEW

File: verification/curator-service/ui/src/components/new-case-form-fields/GenomeSequences.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            <fieldset>
+                <legend>Genome Sequences</legend>
+                <FieldArray name="genomeSequences">
+                    {({ push }): JSX.Element => {
+                        return (
+                            <div>
+                                {values.genomeSequences &&
+                                    values.genomeSequences.map((_, index) => (
+                                        <fieldset
+                                            key={index}
```
Comment (attwad) [inline verification/curator-service/ui/src/components/new-case-form-fields/GenomeSequences.tsx:35]: we shouldn't use the index as key in react lists apparently this is wrong, slows down the app and can show the wrong data :(
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

File: verification/curator-service/ui/src/components/new-case-form-fields/GenomeSequences.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                                                fullWidth
+                                                component={TextField}
+                                            ></Field>
+                                            <Field
+                                                name={`genomeSequences[${index}].notes`}
+                                                type="text"
+                                                label="Notes"
+                                                multiline={true}
+                                                fullWidth
+                                                rows="3"
```
Comment (attwad) [inline verification/curator-service/ui/src/components/new-case-form-fields/GenomeSequences.tsx:82]: you can remove that for now, I know I added that to the other notes but after talking to Shruu the right behavior for textareas is to start on one line and grow as needed, I am not sure how to implement that but I agree that's a better pattern than a fixed number of row.

File: verification/curator-service/ui/src/components/new-case-form-fields/GenomeSequences.tsx (hunk 1 above)
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/new-case-form-fields/GenomeSequences.tsx]: Unfortunate that formik says to use the index in their example :/ https://jaredpalmer.com/formik/docs/api/fieldarray

Done.

File: verification/curator-service/ui/src/components/new-case-form-fields/GenomeSequences.tsx (hunk 2 above)
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/new-case-form-fields/GenomeSequences.tsx]: Done. It does grow when you hit enter.

**Comment by allysonjp715** [comment]
[Finish new case manual entry page](https://trello.com/c/haOgIswr/246-finish-new-case-manual-entry-page)

**Pull request merged**